tests/overlay/commands/open.js

```diff
@@ -172,3 +172,16 @@ QUnit.test(
 		assert.equal(callback.getCall(0).args[0].key, 'overlay');
 	}
 );
+
+QUnit.test(
```

Comment:
The default target of the overlay is the document's body. When enabling the scrollblocker pay attention that the overflow wont be persist after the test run...